optimum/intel/neural_compressor/quantization.py

```diff
@@ -220,10 +220,12 @@ def quantize(
                 " accuracy tolerance has been found. Either the tolerance or the number of trials need to be increased."
             )
         if isinstance(self._original_model.config, PretrainedConfig):
-            original_dtype = self._original_model.config.torch_dtype
-            self._original_model.config.torch_dtype = "int8"
-            self._original_model.config.save_pretrained(save_directory)
-            self._original_model.config.torch_dtype = original_dtype
+            model_config = copy.deepcopy(self._original_model.config)
```

Comment:
Can we modify the original config instead ? (otherwise could create issues if the config is saved later by the user, overwritting this one)

Reply:
But the _original_model is fp32, not int8 model, Maybe we can set _quntized_model with this config.

Reply:
The config from `compressed_model`

Reply:
If backend is IPEX, the JIT model will drop the config attribute, so need copy from original model.

Reply:
set the config to int8 model if not IPEX backend.